contributor/es/04-mechanics-of-contributing-article-es.asciidoc

```diff
@@ -0,0 +1,158 @@
+== Mecánica de la contribución
+
+¿Estás preparado para empezar a contribuir a los proyectos/repositorios de otros equipos?
+¿Quieres reducir tus bloqueos colaborando en vez de gestionar?
+En esta sección se ofrecen consejos prácticos y se destacan los puntos que hay que recordar al realizar una contribución en InnerSource. Permite que el equipo anfitrión y tú tengais una experiencia lo más agradable posible, sentando las bases para más contribuciones y una gran colaboración.
```

Comment:
'Tengan' is more Spanish neutral imo
```suggestion
En esta sección se ofrecen consejos prácticos y se destacan los puntos que hay que recordar al realizar una contribución en InnerSource. Permite que el equipo anfitrión y tú tengan una experiencia lo más agradable posible, sentando las bases para más contribuciones y una gran colaboración.
```

Reply:
We don't seek neutrality (distance to the reader) and _tengan_ would be incoherent with the casual form _tú_.

Reply:
Isn't `tengáis` the correct spelling?

Reply:
tengais is used in some parts. Also i suggested tengan because it refers to both the host team and the reader, so its plural.

Reply:
_tengáis_ is plural as well. It's the casual form instead of the formal one.